acceptance/allocator_terraform/main.tf

```diff
@@ -13,6 +13,7 @@ resource "google_compute_instance" "cockroach" {
   disk {
     image = "${var.gce_image}"
     size = "${var.cockroach_disk_size}" # GB
+    # type    = "pd-ssd" # uncomment for persistent SSD
```

Comment:
It can be plumbed, but I don't see the necessity at this point. Do you think it's worth it?